src/vhost_kern/vdpa.rs

```diff
@@ -253,6 +304,12 @@ mod tests {
         };
     }
 
+    fn kernel_version() -> f32 {
```

Comment:
I honestly always avoid assuming features based on kernel version, because downstream versions don't necessarily have the same features, but since this is just a test it might be okay.